sirepo/nersc.py

```diff
@@ -0,0 +1,44 @@
+# -*- coding: utf-8 -*-
+"""Run Python processes in background
+
+:copyright: Copyright (c) 2016 RadiaSoft LLC.  All Rights Reserved.
+:license: http://www.apache.org/licenses/LICENSE-2.0.html
+"""
+import re
+import subprocess
+from pykern import pkconfig
+from pykern.pkcollections import PKDict
+
+
+_ACCOUNT_NOT_FOUND = "no such fileset"
+
+VALID_TEST_ACCOUNT = "VALID_TEST_ACCOUNT"
+
+
+def assert_project(project):
+    if not project:
+        return ""
+    res = (
+        _test_res(project)
+        if pkconfig.channel_in_internal_test()
+        else subprocess.run(
+            ("hpssquota", project),
+            stdout=subprocess.PIPE,
+            stderr=subprocess.STDOUT,
+            text=True,
+        )
+    )
+    assert not re.search(_ACCOUNT_NOT_FOUND, res.stdout), invalid_project_msg(project)
```

Comment:
We've tended towards:

```py
if re.search(...):
    raise Assertion()
```

Python will strip out `assert` statements when run with certain flags, and this is one we don't want to strip.

However, you probalby want a UserAlert here, because the project is in error, you want to let the user know with a clearly worded error message.

Reply:
[03195d3](https://github.com/radiasoft/sirepo/pull/5493/commits/03195d350b9a432a7a4a3e9444798628b1ac172f)